tools/repos_with_ids.txt

```diff
@@ -122,3 +122,4 @@
 221|github.com/OCA/l10n-ethiopia
 222|github.com/OCA/business-requirement
 223|github.com/OCA/l10n-cambodia
+223|github.com/OCA/infrastructure-dns
```

Comment:
It can't be 223, as this one is for l10n-cambodia. Have you already created the repo in runbot?